### ISSUE TITLE
scrapydweb failed to run on python 3.8

### ISSUE THREAD

**Issue by Aniruddh03** [my8100/scrapydweb#204]
**Describe the bug**
Running scrapydweb locally ,it's failed on startup.

**Logs**
[2022-10-10 15:24:46,295] INFO     in apscheduler.scheduler: Scheduler started
[2022-10-10 15:24:46,335] INFO     in scrapydweb.run: ScrapydWeb version: 1.4.0
[2022-10-10 15:24:46,336] INFO     in scrapydweb.run: Use 'scrapydweb -h' to get help
[2022-10-10 15:24:46,336] INFO     in scrapydweb.run: Main pid: 21725
[2022-10-10 15:24:46,336] DEBUG    in scrapydweb.run: Loading default settings from /home/babadev/.local/lib/python3.8/site-packages/scrapydweb/default_settings.py
Traceback (most recent call last):
  File "/home/babadev/.local/bin/scrapydweb", line 8, in <module>
    sys.exit(main())
  File "/home/babadev/.local/lib/python3.8/site-packages/scrapydweb/run.py", line 32, in main
    app = create_app()
  File "/home/babadev/.local/lib/python3.8/site-packages/scrapydweb/__init__.py", line 81, in create_app
    handle_db(app)
  File "/home/babadev/.local/lib/python3.8/site-packages/scrapydweb/__init__.py", line 123, in handle_db
    db.create_all()
  File "/home/babadev/.local/lib/python3.8/site-packages/flask_sqlalchemy/extension.py", line 868, in create_all
    self._call_for_binds(bind_key, "create_all")
  File "/home/babadev/.local/lib/python3.8/site-packages/flask_sqlalchemy/extension.py", line 839, in _call_for_binds
    engine = self.engines[key]
  File "/home/babadev/.local/lib/python3.8/site-packages/flask_sqlalchemy/extension.py", line 628, in engines
    app = current_app._get_current_object()  # type: ignore[attr-defined]
  File "/home/babadev/.local/lib/python3.8/site-packages/werkzeug/local.py", line 513, in _get_current_object
    raise RuntimeError(unbound_message) from None
RuntimeError: Working outside of application context.
  

**Environment (please complete the following information):**
 - Operating system:Ubuntu 20.04
 - Python version: 3.8
 - ScrapydWeb version: 1.4.0

**Comment by zackfly** [comment]
[https://github.com/my8100/scrapydweb/issues/202](url)
hey bro . I think this cloud be helpful.

**Comment by wmullaney** [comment]
zackfly Link is empty or issue has been deleted.

**Comment by wmullaney** [comment]
Aniruddh03
I know this has been months but I think I know what your issue is. Did you use a virtual environment? If so you either installed part of the process outside the environment or are trying to run outside of the env. Also, I'm sure you've tried many times, logging out and back in. What I have done in the past is I will delete the virtual env folder and recreate it and reinstall everything, log out log back in and then everything works as it should. I couldn't scrapydweb to run. Maybe if I found a docker image but using the documentation as it currently is it is not possible. I have spent hours of my life and my browser has about 30 tabs open on troubleshooting this damn thing. I can't stress how much better the dev world would be if developers prioritized documentation. Poor or out of date documentation destroys open source communities. I am going to see if anyone in here can provide me any help but this might have been a total waste of my time.
